packages/terra-action-header/examples/test-examples/BackActionHeader.example.jsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-alert, import/no-extraneous-dependencies */
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import ActionHeader from '../../lib/ActionHeader';
+
+import messages from '../../translations/en-US.json';
+
+const locale = 'en-US';
+
+export default () => (
+  <IntlProvider locale={locale} messages={messages}>
```

Comment:
You should be able to remove this `IntlProvider` wrapper on these examples. Terra-dev-site now wraps each page in Base which provides this for us. Before these pages were standalone, which is why our current examples wrap with `IntlProvider` or `Base`

This also means you can drop the locale and messages form this example

Reply:
Removed [here](https://github.com/cerner/terra-core/commit/899ef844b8d2ed926f0df690f13e3b97417e8590).